README.md

```diff
@@ -1,10 +1,37 @@
-## lib-kafka
+# LinuxForHealth lib-kafka
 
-Library to create, delete and update kafka topics
+The [asyncio](https://docs.python.org/3/library/asyncio.html) LinuxForHealth [Kafka](https://kafka.apache.org/) client.
 
-The required environment variables before making following API calls:
+## Quickstart
 
-KAFKA_BROKER_CONFIG_FILE: The config file path for broker configuration. Default value is "/var/app/config/kafka.env"
+### Pre-requisites
+
+* Python >= 3.8
+* librdkafka >= 1.6.0
+
+LinuxForHealth lib-kafka is built on Confluent's Kafka Python client, which in turn relies on a "native" C dependency, librdkafka.
+librdkafka is included in Confluent's Kafka-Python wheel distribution for some platforms. For other platforms it needs to be installed prior to use.
+Please refer to [Confluent's documentation](https://github.com/confluentinc/confluent-kafka-python#prerequisites) for additional information.
+
+### Project Setup and Validation
+```shell
+git clone https://github.com/LinuxForHealth/lib-kafka
+cd lib-kafka
+
+python3 -m venv venv && source venv/bin/activate && pip install --upgrade pip setuptools 
+pip install -e .[test]
```

Comment:
If `zsh` is your shell, which is now the default for OS X, you will have to use double quotes here
```shell
pip install -e ".[test]"
```